doc/source/tutorials/tutorial-level-1-2-3.rst

```diff
@@ -260,12 +260,14 @@ Before running code, you need to set the ``PYTHONPATH`` environment variable to
 
      .. code-block:: bash
 
-          $ python proj-one/proj_one_code.py
+          $ cd proj-one
```

Comment:
Right. People are more likely to visit the repo and run the code.

Reply:
repo here I mean the individual project directory

Reply:
I just felt that it was more pedagogical and easier to follow.  Once the reader gets the logic they can put 2 and 2 together ny themselves and realize they could have run it your way.  But I just felt this was easier to follow.